docs/dependencies/metadata-storage.md

```diff
@@ -23,50 +23,70 @@ title: "Metadata storage"
   -->
 
 
-The Metadata Storage is an external dependency of Apache Druid. Druid uses it to store
-various metadata about the system, but not to store the actual data. There are
-a number of tables used for various purposes described below.
+Apache Druid relies on an external dependency for metadata storage.
+Druid uses the metadata store to house various metadata about the system, but not to store the actual data.
+The metadata store retains all metadata essential for a Druid cluster to work.
+
+The metadata store includes the following:
+- Segments records
+- Rule records
+- Runtime configuration objects
```

Comment:
What are Runtime configuration objects?

Reply:
That's how the [config table](https://druid.apache.org/docs/latest/dependencies/metadata-storage.html#config-table) section is described. I'll rephrase to match the subheading text.

Reply:
```suggestion
- Configuration records
```